source/brailleDisplayDrivers/handyTech.py

```diff
@@ -581,10 +602,47 @@ def __init__(self, port="auto"):
 			self._dev.close()
 
 		else:
+			wx.CallAfter(self.destroy_message_window)
 			raise RuntimeError("No Handy Tech display found")
 
+	def create_message_window(self):
+		try:
+			self._messageWindow = InvisibleDriverWindow(self)
+		except:
+			log.debugWarning("", exc_info=True)
+	def destroy_message_window(self):
+		try:
+			self._messageWindow.destroy()
+		except:
+			log.debugWarning("", exc_info=True)
+
+	def go_to_sleep(self):
+		self._sleepcounter += 1
+		if self._dev is not None:
+			self._dev.close()
+			self._dev = None
+			time.sleep(self.timeout)
+	def wake_up(self):
+		if self._sleepcounter > 0:
+			self._sleepcounter -= 1
+		if self._sleepcounter > 0: # Still not zero after decrementing
+			return
+		# Might throw if device no longer exists.
+		# We leave it to autodetection to grab it when it reappears.
+		if self.isHidSerial:
+			# This is either the standalone HID adapter cable for older displays,
+			# or an older display with a HID - serial adapter built in
+			self._dev = hwIo.Hid(self.port, onReceive=self._hidSerialOnReceive)
+			# Send a flush to open the serial channel
+			self._dev.write(HT_HID_RPT_InCommand + HT_HID_CMD_FlushBuffers)
+		elif self.isHid:
+			self._dev = hwIo.Hid(self.port, onReceive=self._hidOnReceive)
+		else:
+			self._dev = hwIo.Serial(self.port, baudrate=BAUD_RATE, parity=PARITY,
+				timeout=self.timeout, writeTimeout=self.timeout, onReceive=self._serialOnReceive)

```

Comment:
Could you ensure that there is at least one empty line between every new method definition?